pkg/dynamicaccess/accesslogic_test.go

```diff
@@ -88,19 +88,14 @@ func TestGet_Error(t *testing.T) {
 
 	encryptedRef, _ := al.EncryptRef(ref, &id0.PublicKey, swarm.NewAddress([]byte(expectedRef)))
 
-	r, err := al.Get(swarm.RandAddress(t), encryptedRef, &id0.PublicKey)
-	if err == nil {
+	r, err := al.DecryptRef(swarm.RandAddress(t), encryptedRef, &id0.PublicKey)
+	if err != nil {
 		t.Logf("r: %s", r.String())
 		t.Errorf("Get should give back encrypted access key not found error!")
 	}
 
-	refTwo, _ := al.Get(swarm.RandAddress(t), swarm.EmptyAddress, &id0.PublicKey)
-	if swarm.EmptyAddress.Compare(refTwo) != 0 {
-		t.Errorf("Get should give back empty string if encrypted ref not provided!")
-	}
-
-	_, err = al.Get(swarm.RandAddress(t), encryptedRef, nil)
-	if err == nil {
+	_, err = al.DecryptRef(swarm.RandAddress(t), encryptedRef, nil)
+	if err != nil {
```

Comment:
I think it should be err == nil

Reply:
We deleted the whole function. It will give back a result for any encrypted Swarm address, even if it does not exist.

Reply:
I can make a change in session.Key so that it returns an error if the pubkey is invalid